docs/sources/configure-client/language-sdks/dotnet.md

```diff
@@ -9,24 +9,53 @@ aliases:
 
 # .NET
 
-## Supported platforms
+Our .NET Profiler is a powerful tool designed to enhance the performance analysis and optimization of .NET applications. It seamlessly integrates with Pyroscope, offering real-time insights into the resource usage and bottlenecks within your .NET codebase. This integration empowers developers to pinpoint inefficiencies, improve application speed, and ensure resource-efficient operation.
 
-| Spy Name     | Type         | Linux | macOS | Windows | Docker |
-|:------------:|:------------:|:-----:|:-----:|:-------:|:------:|
-| dotnetspy    | `embedded`   |   ✅  |       |         |        |
+### Supported Profiling Types
 
-## Supported .NET versions:
- - .NET 6.0
- - .NET 7.0
+The .NET Profiler supports the following profiling types:
 
-## How to add .NET profiling to your application
+* CPU
+* Wall time
+* Allocations
+* Lock contention
+* Exceptions
+
+### Compatibility
+
+Our .NET profiler works with the following .NET versions:
+* .NET 6
+* .NET 7
+* .NET 8
+
+
+## Before you Begin
+
+### Set Up a Pyroscope Server
+
+To capture and analyze profiling data, set up a Pyroscope server. This can be:
+
+* A **local server** for development, or
+* A **remote server** for production use.
+
+For installation instructions, see our [Get Started]({{< relref "../../get-started" >}}) guide.
+
+
+### Using Grafana Cloud Profiles
```

Comment:
If the above is correct, then we don't need Using in the heading here. 
```suggestion
### Grafana Cloud Profiles
```

Reply:
Why is this better to remove the using here? I prefer the more clear title over the generic one. If it's just "grafana cloud profiles", then I'm left wondering -- _what about_ grafana cloud profiles? also for seo purposes someone would more likely search "how to use" or "using"

Reply:
This section is before you begin -- which discusses requirements for using the procedures on the page. "Using" or "Use" implies there are steps to be followed, which we're not using. We're stating you have to have either Pyroscope server OR Grafana Cloud Profiles to use the procedure. 

For more information: https://grafana.com/docs/writers-toolkit/structure/topic-types/task/
Also, stylistically we use the imperative instead of a gerund in headings.

Reply:
Actually, if I was going to re-do this, I'd make it much shorter. 

To capture and analyze profiling data, you need either a hosted Pyroscope OSS server or hosted Pryoscope instance with Grafana Cloud Profiles](/products/cloud/profiles-for-continuous-profiling/) (requires a free Grafana Cloud account). 

The Pyroscope server can be a local server for development or a remote server for production use.f

and that's it. Technically, the description of Grafana Cloud Profiles is not needed to understand how to do the procedure. People can use the link if they want to learn more about it. It's out of scope for the topic.